src/onnx/parse_conv_transpose.cpp

```diff
@@ -80,14 +78,21 @@ struct parse_deconvolution : op_parser<parse_deconvolution>
                                std::back_inserter(values["padding"]),
                                [](auto pad_val) { return pad_val; });
             }
+            else if(l0->get_shape().dynamic())
+            {
+                MIGRAPHX_THROW("PARSE_CONV_TRANSPOSE: asymmetric padding (padding_L != padding_R) "
+                               "not supported with dynamic shapes");
+            }
```

Comment:
Is it missing setting padding values for the asymmetrical + static shape case? 
I don't see it setting padding values at all for that case.